docs/source/host/maintain-domain/update-software.rst

```diff
@@ -22,21 +22,21 @@ Both of these will result in an error message "Protocol Mismatch".
 Upgrade Your Domain Server
 ------------------------------------------
 
-Because all of Vircadia's users are encouraged to update to the latest release, it is highly likely that the majority of your visitors are running the newest version of Vircadia. Therefore, if there has been a protocol change (check the :doc:`Release Notes <../../release-notes>` for your version), we strongly recommend that you upgrade your domain server to the newest release.
+Because all of Overte's users are encouraged to update to the latest release, it is highly likely that the majority of your visitors are running the newest version of Overte. Therefore, if there has been a protocol change (check the :doc:`Release Notes <../../release-notes>` for your version), we strongly recommend that you upgrade your domain server to the newest release.
 
 To update a local server on Windows:
 
-1. Download the latest `Client + Sandbox installer <https://vircadia.com/download-vircadia/#server>`_ from Vircadia's website or download the upgrade when prompted to on your server.
+1. Download the latest `Client + Sandbox installer <https://vircadia.com/download-vircadia/#server>`_ from Overte's website or download the upgrade when prompted to on your server.
```

Comment:
Can you add a note to the top that tells us that we need to come back and update this?
Something like:
```
.. warning::
    This document is slightly outdated.
    FIXME: download link is outdated

```

Reply:
Done. Thanks for noticing this!